coding/python_style_guide.rst

```diff
@@ -620,6 +643,40 @@ For more information see the online book `Supporting Python 3 <http://python3por
 
 .. _futurize: http://python-future.org/futurize.html
 
+Supporting Python 2.7 and 3.x simultaneously
+---------------------------------------------
+
+The ``future`` package MUST be used to provide compatibility functionality with Python 2.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We use the `future <http://python-future.org/>`_ package to provide a means for writing code using Python 3 idioms that will also work on Python 2.7.
+Details on the process for migrating a 2.7 codebase to support both versions can be found in `SQR-014 <https://sqr-014.lsst.io>`_.
+
+.. _style-guide-py-future-absolute-import:
+.. _style-guide-py-future-division:
+.. _style-guide-py-print:
+
+``__future__`` MUST be used to import Python 3 behavior in all files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Code that is to be used on Python 2.7 and 3.x should import ``division``, ``print_function`` and ``absolute_import`` from the :mod:`__future__` package.
```

Comment:
"MUST" vs. "should" is confusing.  Use "must" here but qualify with "where the related functionality is used"?